doc/new_user.rst

```diff
@@ -4,7 +4,11 @@ New user basics
 Set up teuthology
 -----------------
 
-#. Log in to teuthology.front.sepia.ceph.com
+#. Request access via the process described at:
+   https://github.com/ceph/sepia/blob/master/doc/adding_users.rst
```

Comment:
I just took a look at http://ceph.github.io/sepia/ - and yeah, at some point we'll want to revamp these docs to have better formatting, I agree.